cleverhans/model.py

```diff
@@ -0,0 +1,107 @@
+from abc import ABCMeta
+
+
+class Model(object):
```

Comment:
Alex and I just talked in real life and we agreed to not include the model loss per the discussion we had on cleverhans-dev